io.openems.edge.controller.ess.timeframe/src/io/openems/edge/controller/ess/timeframe/Mode.java

```diff
@@ -0,0 +1,5 @@
+package io.openems.edge.controller.ess.timeframe;
+
+public enum Mode {
+	MANUAL, OFF;
```

Comment:
I guess "ON", "AUTO", "OFF" would be better as "ON" charges as it should do as configured. etc.